description: wrong define was used, causing errors building code intended for MSVC
author: Michael Gilbert <mgilbert@debian.org>

--- a/third_party/angle/src/common/debug.h
+++ b/third_party/angle/src/common/debug.h
@@ -248,7 +248,7 @@ std::ostream &FmtHex(std::ostream &os, T
 #    define EVENT(message, ...) (void(0))
 #endif
 
-#if defined(COMPILER_GCC) || defined(__clang__)
+#if defined(__GNUC__) || defined(__clang__)
 #    define ANGLE_CRASH() __builtin_trap()
 #else
 #    define ANGLE_CRASH() ((void)(*(volatile char *)0 = 0)), __assume(0)
